docs/how-to-guides/running-autoware-without-cuda.md

```diff
@@ -0,0 +1,34 @@
+# Running Autoware without CUDA
+
+Two following algorithms in Autoware Universe depend on a CUDA environment.
+
+- 2D/3D object detection
+- traffic light detection
+
+However, it is still possible to run Autoware in the environment without CUDA.
+The following subsections briefly explain how to run these algorithms in such an environment.
+
+!!! warning
+
+    It is highly recommended to prepare a CUDA environment to achieve better performance.
+
+## Running 2D/3D object detection without CUDA
+
+As for the object detection system in Autoware, the following three packages require the CUDA environment:
+
+- `lidar_centerpoint`
+- `lidar_apollo_instance_segmentation`
+- `tensorrt_yolo`
+
+Instead of these modules, you can use the `euclidean_cluster` module, which is a 3D object detection algorithm that does not require a CUDA environment (refer to [the readme](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/euclidean_cluster) for detail).
```

Comment:
Reworded/rephrased. Is it correct to say that the two lidar* modules related to LiDAR-based detection whilst `tensorrt_yolo` is used for camera-based detection? If so, then is it also correct to say that `euclidean_cluster` can onyl be used instead of the lidar-related modules?

```suggestion
Instead of these modules, you can use the `euclidean_cluster` module which contains a 3D object detection algorithm that does not require CUDA. For more details, refer to the [`euclidean_cluster` module's README file](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/euclidean_cluster).
```